src/axiselements.jl

```diff
@@ -13,12 +14,14 @@ end
 Base.push!(plot::Plot, element) = (push!(plot.elements, element); plot)
 Base.append!(plot::Plot, element) = (append!(plot.elements, element); plot)
 
-function Plot(elements::AbstractVector, args::Vararg{PGFOption}; incremental = true, label = nothing)
-    Plot(elements, dictify(args), label, incremental, false)
+function Plot(elements::AbstractVector, args::Vararg{PGFOption}; incremental =
```

Comment:
This line breaking seems odd to me.

Reply:
I take advantage of Julia's flexibility with newlines. I hard-wrap at 80 characters. It's definitely a bad line break. Feel free to change it.